how-to-release.md

```diff
@@ -8,41 +8,49 @@ You may obtain a copy of the License at
+# Preparing the Branch
 
-* Bug-fix release: This is a minor release version over an existing release branch
-* Feature release or non-backward-compatible release: This is a change to either the first or the middle digit, and it requires a new release branch.
+Preparing the branch consists of making the necessary changes to the branch we will be working on as part of releasing. Following are the possible two situations:
 
-## Bug-fix release
+ -  **Bug fix release**: This is a minor release version over an existing release branch.
+ -  **Feature release or non-backward-compatible release**: This is a change to either the first or the middle digit, and it requires a new release branch.
+
+## Bug Fix Release
+
+For bug fix release, no new branch is required. First identify the branch we will be working on. It will be named `rX.Y`, (e.g., `r0.2`). The preparation consists of:
 
-If this is a bug-fix release, then there is no need to create a new branch. First identify the branch you'll be working on. It will be named `rX.Y`, e.g., `r0.2`. The preparation consists of:
 1. Changing `connectorVersion` in `gradle.properties` from `X.Y.Z-SNAPSHOT` to `X.Y.Z`. For example, if the current value is `connectorVersion=0.2.1-SNAPSHOT`, then change it to `connectorVersion=0.2.1`.
-2. Merge this change
-3. Tag the commit with `vX.Y.Z-rc0`. For example, `v0.2.1-rc0`.
+2. Merge this change.
+3. Tag the commit with `vX.Y.Z-rc0`. (For example, `v0.2.1-rc0`).
 
-A couple of observations about step 3:
+Please note the following when performing step 3:
 1. There are two ways to tag:
-    *  *Via the command line*:
-    ```
+    -  **Via the command line**:
+
+        ```
        > git checkout rX.Y
        > git tag vX.Y.Z-rc0
        > git push upstream vX.Y.Z-rc0
-    ```
-    Make sure you have your `upstream` set up correctly
-    
-    *  *Via GitHub releases*: when creating a release candidate, GitHub automatically creates the tag for you in the case one does not exist yet. We will take more about release candidates in a minute.
+        ```
+    Ensure that our `upstream` is set up correctly.
+
+    -  **Via GitHub releases**: When creating a release candidate, Github automatically creates the tag if one doesn't exist. This is discussed in the [release candidate](#pushing-arelease-candidate) section.
 
-2. It is possible that a release candidate is problematic and we need to do a new release candidate. In this case, we need to repeat this tagging step as many times as needed. Note that when creating a new release candidate tag, we do not need to update the Connector version.
+2. In situations, sometimes the release candidate is problematic when we do a new release candidate. Hence, this can be handled by repeating the tagging step multiple times until it is created.
 
-## Major release
+**Note**: Do not update the Connector version, when creating a new release candidate tag.

```

Comment:
Can we replace line 46:48 with the original statement.

> It is possible that a release candidate is problematic and we need to do a new release candidate. In this case, we need to repeat this tagging step as many times as needed. Note that when creating a new release candidate tag, we do not need to update the Connector version.

Reply:
Accepted suggestion